packages/hub/src/interfaces.ts

```diff
@@ -20,9 +21,23 @@ export interface Carrier {
   __SENTRY__?: {
     hub?: Hub;
     /**
-     * These are extension methods for the hub, the current instance of the hub will be bound to it
+     * Extra Hub properties injected by various SDKs
      */
-    // eslint-disable-next-line @typescript-eslint/ban-types
-    extensions?: { [key: string]: Function };
+    extensions?: {
+      /** Hack to prevent bundlers from breaking our usage of the domain package in the cross-platform Hub package */
+      domain?: typeof domain & {
+        /**
+         * The currently active domain. This is part of the domain package, but for some reason not declared in the
+         * package's typedef.
+         */
+        active?: domain.Domain;
+      };
+    } & {
+      /** Extension methods for the hub, which are bound to the current Hub instance */
+      // eslint-disable-next-line @typescript-eslint/ban-types
+      [key: string]: Function;
+    };
```

Comment:
H: Are we sure that doesn't break in Browser environments?

Reply:
I will check again, for sure, but the change here isn't a behavior change but merely a TS one. We've always put the domain there, it's just that before we didn't include it in the type so we had to do a lot of `eslint-disable @no-unsafe-member-access` and `eslint-disable @no-explicit-any`. My understanding is that TS stuff all gets stripped?

Anyway, will check and update here.

Reply:
UPDATE: Seems to build just fine, with no special accommodation in my set up.

Reply:
@lobsterkatie It seems like these changes broke `@sentry/browser` for TS users by requiring node types 😕 issue: https://github.com/getsentry/sentry-javascript/issues/2927

Reply:
See #3050 for a fix.